### DISCUSSION TITLE
When generating `summary` add `<exception>`  for every exception thrown in a method

### DISCUSSION THREAD

**Discussion by maxkoshevoi** [dotnet/csharplang#4850] [Language Ideas] [answered]
This is related to Visual Studio (didn't find a better place to post this).

**Idea**
When user types `///` above a method, `summary` comment is generated with place for method and parameter descriptions.
Suggestion is to also automatically add place for `exception` description (if the method explicitly throws some exception(s)). 

**Motivation**
It's useful to see what exceptions can be thrown from a method and for what reason, but most developers don't even know that it's possible to communicate that information to method's consumer.
This will familiarize developers with existence of `<exception>` tag and facilitate it's adoption. So far it's mostly used by framework methods (that's how I found out that it exists).

**Comment by 333fred** [comment, marked as answer]
The correct place to open this is on dotnet/roslyn as a feature request.

**Reply by maxkoshevoi** [reply]
There's no template for feature request there:

![image](https://user-images.githubusercontent.com/6609929/122684480-455c8b00-d20e-11eb-83a6-ab6571f7f8de.png)

**Reply by 333fred** [reply]
Just choose bug report and modify the template.

**Reply by maxkoshevoi** [reply]
Moved to roslyn, thank you https://github.com/dotnet/roslyn/issues/54245
